out/CMakeCache.txt

```diff
@@ -0,0 +1,534 @@
+# This is the CMakeCache file.
```

Comment:
Has this cache file been modified at all? If so, should we check these into a cmake/caches dir instead?

Reply:
eek, the `out` directory shouldn't be there!